applications/GeoMechanicsApplication/tests/test_settlement_workflow/MaterialParameters.json

```diff
@@ -13,7 +13,7 @@
               "DENSITY_SOLID"            :  2.65,
               "DENSITY_WATER"            :  1.0,
               "POROSITY"                 :  0.3,
-              "BULK_MODULUS_SOLID"       :  1.0e9,
+              "BULK_MODULUS_SOLID"       :  35.0e6,
```

Comment:
These are changes proposed by Marjan and Wijtze Pieter for the test case

Reply:
Looks good to me.